### DISCUSSION TITLE
Welcome to elm-review Discussions!

### DISCUSSION THREAD

**Discussion by jfmengels** [jfmengels/elm-review#23] [General]
(Note: I still have no clue how this space will work out for us, so I'm enabling it as a test)

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sparksp** [comment]
I'm Phill and I have authored a number of elm-review rules (both within @jfmengels packages and my own). I'm a big fan of TDD.

**Comment by jfmengels** [comment]
I'm Jeroen and I'm the author of the `elm-review` package and CLI, and many of the published packages. I hope you enjoy the tool :blush:
